ast/parser_test.go

```diff
@@ -855,6 +855,12 @@ func TestEvery(t *testing.T) {
 		"unexpected ident token: expected \\n or ; or }\n\tevery x\n", // this asserts that the tail of the error message doesn't contain a hint
 	)
 
+	assertParseErrorContains(t, "x in ... usage is hinted properly", `
+	p {
+		x in {"foo": "bar"} { is_string(x); is_string(y) }
```

Comment:
This example is also not valid rego, let's do

```suggestion
		"bar" in {"foo": "bar"}
```

(see comment above)

Reply:
Thank you for your help, I have made modifications according to the above suggestions and found a more suitable method, please review again. Thanks!